forwards_paper.tex

```diff
@@ -289,11 +299,6 @@ \subsection*{Working with a tree sequence through tables}
         The right-hand panels show the specific encoding
         of this tree sequence in the four tables (nodes, edges, sites and mutations)
         defined by \msprime.
```

Comment:
I re-did this figure in inkscape (a good activity for a long plane flight).  What do you think?  Adjustments?  I wish I could do this in a code-ey way, like asy... maybe I will have to learn tikz...

Reply:
Looks good I think... Thoughts:

- The trees look a bit crowded to me. Maybe the node circles could be a little smaller or the mutations stars a little smaller?
- I see the motivations for chopping off the "state" from ancestral and derived, but maybe this is a bit confusing overall? I think it's probably better as you have it, but I thought I should raise the issue.
- The intervals section is a bit confusing for me. I see what you've done now, but I didn't get it at all initially. What if we shrunk the edge labels and put them on top of the line segments?
- Alignment looks a bit off in places to me (e.g., centering of node labels).

But it's definitely an improvement, much simpler  and less fussy. I think after some minor tweaks  it's a winner.

Reply:
Done those tweaks; thanks!  Pushing now.  Happy to do more.

Reply:
Superb! Thanks for doing this @petrelharp, it looks way better than my version!